### PR TITLE
MP-5145 Cache Strategy Static

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/MP-5145

### CODE REVIEW

File: docs/marketplace/dev/front-end/ui-components-library/datasources/index.md [hunk 1]
```diff
@@ -107,4 +107,4 @@ There are a few common Datasources that are available in UI library as separate
 - [inline](/docs/marketplace/dev/front-end/ui-components-library/datasources/datasource-inline.html) — Allows passing data along with the configuration of the Datasource.
 - [inline.table](/docs/marketplace/dev/front-end/ui-components-library/datasources/datasource-inline-table.html) — Allows passing transformed for the table format data along with the configuration of the Datasource
 - [http](/docs/marketplace/dev/front-end/ui-components-library/datasources/datasource-http.html) — Allows fetching data from URL via HTTP configured in the configuration of the Datasource. 
-HTTP Datasource supports caching strategy (see [Cache](/docs/marketplace/dev/front-end/ui-components-library/cache.html)) that may be configured via config and used before the request is made when applicable.
\ No newline at end of file
+HTTP Datasource supports caching strategy (see [Cache](/docs/marketplace/dev/front-end/ui-components-library/cache.html)) that may be configured via config and used before the request is made when applicable.
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/datasources/index.md:110]:
```suggestion
HTTP Datasource supports caching strategy (see [Cache](/docs/marketplace/dev/front-end/ui-components-library/cache/)) that may be configured via config and used before the request is made when applicable.
```

File: docs/marketplace/dev/front-end/ui-components-library/cache/static.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+`expiresIn` - represents a duration as string where components are separated by a space.  
+
+```html
+<spy-select
+  [datasource]="{
+    type: 'http',
+    ...,
+    cache: {
+      type: 'static',
+      expiresIn: '1m 10d 3h',
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/cache/static.md:25]: It is not clear how the minutes should be added.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/front-end/ui-components-library/cache/static.md (hunk 2 above)
Comment (danilmoro3) [reply docs/marketplace/dev/front-end/ui-components-library/cache/static.md:25]: Added examples to the interface

File: docs/marketplace/dev/front-end/ui-components-library/cache/static.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+></spy-select>
+```
+
+## Interfaces
+
+Below you can find interfaces for the Cache Strategy Static.
+
+```ts
+/**
+ * Represents a duration as string where components are separated by a space
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/ui-components-library/cache/static.md:37]:
```suggestion
 * Represents a duration as a string where components are separated by a space
```

**Review by FRITS-Kh — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**